Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -20,10 +20,11 @@ class CurrentUserUpdater: Worker {
     ///   - completion: Called when user is successfuly updated, or with error.
     func updateUserData(
         currentUserId: UserId,
-        name: String? = nil,
-        imageURL: URL? = nil,
-        privacySettings: UserPrivacySettings? = nil,
-        userExtraData: [String: RawJSON]? = nil,
+        name: String?,
+        imageURL: URL?,
+        privacySettings: UserPrivacySettings?,
+        role: UserRole?,
+        userExtraData: [String: RawJSON]?,
```

Comment:
Removed default values for since that is why we forgot to add privacySettings to CurrentUser's method.